instructor/common.py

```diff
@@ -1904,3 +1955,11 @@ def is_long(s):
         return True
     except ValueError:
         return False
+
+
+@register.filter
+def get_item(dictionary, key):
+    if type(dictionary) is dict:
+        return dictionary.get(key)
+    else:
+        return getattr(dictionary, key)
```

Comment:
This will raise an exception if `key` doesn't exist in `dictionary. Can you change to `getattr(dictionary, key, None)` to be consistent with the other case above?